extractor/src/main/java/org/schabi/newpipe/extractor/utils/AbbreviationHashMap.java

```diff
@@ -1,13 +1,36 @@
-package org.schabi.newpipe.extractor.services.youtube.extractors;
+package org.schabi.newpipe.extractor.utils;
 
 import java.util.HashMap;
 
-public class YoutubeAbbreviationSubCountMap {
+
+/**
+ * Map matching abbreviations with their English equivalents
```

Comment:
The class only contains maps

Reply:
You wanna rename it to what?

Reply:
Sorry, utils is the wrong package here. Should be placed in localization and probably called AbbreviationHelper. @Stypox Do you have an idea for a proper name?